src/ale.cpp

```diff
@@ -367,7 +367,7 @@ namespace ale {
      // Create a Python tuple to hold the arguments to the method.
      PyObject *pArgs = PyTuple_New(3);
      if(!pArgs) {
-       throw runtime_error(getPyTraceback());
+       throw runtime_error("whoops");
```

Comment:
May want to add a better error here. Not sure if the traceback is still viable but that may be something good to replace this with.